### PR TITLE
Set up VSCode debug configuration with Google Chrome

### PR DESCRIPTION
**Pull request by nickImbirev**

### CODE REVIEW

File: clients/browser/webpack.dev.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const path = require('path');
```
Comment (nickImbirev) [inline clients/browser/webpack.dev.js:1]: I propose to add something new in this file only when necessary, to keep it super simple...

**Comment by imbireva** [comment]
The same question: what is this and how can I use it?.

**Review by imbireva — APPROVED** [review]

**Comment by nickImbirev** [comment]
> What is this and how can I use it?

Yeah, sorry, I forgot to clarify the usage of these tools in some README file, I added some tips in this [commit](https://github.com/nickImbirev/cards-service/pull/6/commits/bee40763eb8e369ef9269b5c8f46c40b33f030d5).

Hope it helps

**Comment by imbireva** [comment]
> Yeah, sorry, I forgot to clarify the usage of these tools in some README file, I added some tips in this [commit](https://github.com/nickImbirev/cards-service/pull/6/commits/bee40763eb8e369ef9269b5c8f46c40b33f030d5).
Hope it helps

Yeah, don't say sorry, it's okay, you do a lot of good things to help me!

Thank you!

**Pull request merged**